lms/templates/dashboard/_dashboard_xseries_info.html

```diff
@@ -3,34 +3,36 @@
-            </p>
-            <p class="message-copy">
-                ${_("This course is 1 of {course_count} courses in the {link_start}{program_display_name}{link_end} {program_category}.").format(
-                    course_count=course_program_info['course_count'],
-                    link_start='<a href="{}">'.format(course_program_info['program_marketing_url']),
-                    link_end='</a>',
-                    program_display_name=course_program_info['display_name'],
-                    program_category=course_program_info['display_category'],
-                )}
-            </p>
+%for program_data in course_program_info.get('course_program_list', {}):
+    <div class="message message-status is-shown credit-message">
+        <div class="xseries-action">
+            <div class="message-copy xseries-msg">
+                <p>
+                    <b class="message-copy-bold">${_("{category} Program: Interested in more courses in this subject?").format(category=course_program_info.get('display_category', ''))}</b>
+                </p>
+                <p class="message-copy">
+                    ${_("This course is 1 of {course_count} courses in the {link_start}{program_display_name}{link_end} {program_category}.").format(
+                        course_count=program_data['course_count'],
+                        link_start='<a href="{}">'.format(program_data['program_marketing_url']),
+                        link_end='</a>',
+                        program_display_name=program_data['display_name'],
+                        program_category=course_program_info.get('display_category', ''),
+                    )}
+                </p>
 
+            </div>
+            <%
+                xseries_btn_class = "xseries-border-btn"
+                if enrollment_mode == "verified":
+                    xseries_btn_class = "xseries-base-btn";
+            %>
+            <a class="btn ${xseries_btn_class}" href="${program_data['program_marketing_url']}" target="_blank"
+               data-program-id="${program_data['program_id']}" >
+                <i class="action-xseries-icon"></i>
+                <span>
+                    ${_("View {category} Details").format(category=course_program_info.get('display_category', ''))}
+                </span>
+            </a>
         </div>
-        <%
-            xseries_btn_class = "xseries-border-btn"
-            if enrollment_mode == "verified":
-                xseries_btn_class = "xseries-base-btn";
-        %>
-        <a class="btn ${xseries_btn_class}" href="${course_program_info['program_marketing_url']}" target="_blank"
-           data-program-id="${course_program_info['program_id']}" >
-            <i class="action-xseries-icon"></i>
-            <span>
-                ${_("View {category} Details").format(category=course_program_info['display_category'])}
-            </span>
-        </a>
     </div>
-</div>
+%endfor
```

Comment:
Do we follow the same newline convention for python as we do for Mako templates? I'll definitely make the change, but just wondering if this is expected in the future as well?